charts/scalardb-mcp-server/values.yaml

```diff
@@ -0,0 +1,256 @@
+    scalar.db.sql.cluster_mode.contact_points=${scalar.db.contact_points}
+    scalar.db.sql.cluster_mode.contact_port=${scalar.db.contact_port}
+
+    # ========================================
+    # Logging Configuration
+    # ========================================
+    scalar.mcp.db.server.logging.level=${SCALAR_MCP_DB_SERVER_LOGGING_LEVEL:DEBUG}
+    scalar.mcp.db.server.logging.file.name=${SCALAR_MCP_DB_SERVER_LOGGING_FILE_NAME:}
+    logging.level.com.scalar.scalardb.mcp=${scalar.mcp.db.server.logging.level}
+    logging.file.name=${scalar.mcp.db.server.logging.file.name}
+    logging.threshold.console=on
+
+  # -- Environment variables for the container. Use this to inject secrets and configuration.
+  # @default -- No environment variables are set by default.
+  env: []
+    # Required secrets - MUST be provided for production:
+    # - name: SCALAR_MCP_DB_OAUTH_SESSION_CACHE_ENCRYPTION_KEY
+    #   valueFrom:
+    #     secretKeyRef:
+    #       name: scalardb-mcp-server-secrets
+    #       key: session-encryption-key
+    # - name: SCALAR_MCP_DB_OAUTH_SECURITY_JWT_PRIVATE_KEY
+    #   valueFrom:
+    #     secretKeyRef:
+    #       name: scalardb-mcp-server-secrets
+    #       key: jwt-private-key
+    # - name: SCALAR_MCP_DB_OAUTH_SECURITY_JWT_PUBLIC_KEY
+    #   valueFrom:
+    #     secretKeyRef:
+    #       name: scalardb-mcp-server-secrets
+    #       key: jwt-public-key
+    #
+    # Optional configuration overrides:
+    # - name: SCALAR_MCP_DB_OAUTH_ISSUER_URL
+    #   value: "https://mcp.example.com"
+    # - name: SCALAR_DB_CONTACT_POINTS
+    #   value: "indirect:scalardb-cluster-envoy.scalardb.svc.cluster.local"
+    # - name: SCALAR_MCP_DB_SERVER_LOGGING_LEVEL
+    #   value: "DEBUG"
+
+  service:
+    # -- Service type.
+    type: ClusterIP
+    # -- Service port.
+    port: 8080
+    # -- Container target port.
+    targetPort: 8080
+    # -- Protocol for the service.
+    protocol: TCP
+    # -- Service annotations.
+    annotations: {}
+
+  # -- Persistence configuration for SQLite database.
+  persistence:
+    # -- Enable persistence using PVC.
+    enabled: true
+    # -- Use an existing PVC instead of creating a new one.
+    existingClaim: ""
+    # -- Storage class for dynamic provisioning. Leave empty for default.
+    storageClass: ""
+    # -- Access mode for the PVC.
+    accessMode: ReadWriteOnce
+    # -- Size of the PVC.
+    size: 1Gi
+    # -- Annotations for the PVC.
+    annotations: {}
+    # -- Mount path for the persistent volume.
+    mountPath: /scalardb-mcp-server/data
+
+# -- Ingress configuration with TLS termination support.
+ingress:
+  # -- Enable ingress.
+  enabled: false
+  # -- Ingress class name (e.g., nginx, alb, azure/application-gateway).
+  className: ""
+  # -- Ingress annotations.
+  annotations: {}
+    # Example annotations for nginx:
+    # nginx.ingress.kubernetes.io/ssl-redirect: "true"
+    # nginx.ingress.kubernetes.io/backend-protocol: "HTTP"
```

Comment:
Ditto.

I think it would be better to remove example configurations about Ingress NGINX Controller.